modules/datahandling/userrequests.go

```diff
@@ -234,9 +235,43 @@ func (f userProjectsRequest) process(db dbfs.DBFS) ([]dhClosure, error) {
 			Status: messages.StatusPartialfail,
 			Tag:    f.Tag,
 			Data: struct {
-				Projects []dbfs.ProjectMeta
+				Projects []projectLookupResult
 			}{
-				Projects: projects,
+				Projects: nil,
+			},
+		}.Wrap()
+		return []dhClosure{toSenderClosure{msg: res}}, err
+	}
+
+	resultData := make([]projectLookupResult, len(projects))
+	var errOut error
+	i := 0
+	for _, project := range projects {
+		lookupResult, err := projectLookup(f.SenderID, project.ProjectID, db)
+
+		if err != nil {
+			utils.LogError("Project lookup error", err, utils.LogFields{
+					"Resource": f.Resource,
+					"Method": f.Method,
+					"SenderID": f.SenderID,
+					"ProjectID": project.ProjectID,
+			})
+			err = errOut
```

Comment:
this line should be `errOut = err`, otherwise you're just throwing err away and you never set errOut